joomla/joomla-cms
# Add PDO PostgreSQL support and deprecate native PostgreSQL

Pull Request for Issue #20272

### Summary of Changes

- Adds a database driver for PDO PostgreSQL support
- Deprecates the database driver supporting PHP's `ext/pgsql`
- Updates relevant 4.0 compatibility checks

### Testing Instructions

In an environment with PostgreSQL database support, make sure PHP has PDO support enabled with the PostgreSQL engine available (if you can already run native `ext/pgsql` then you should only need to enable the PDO support if it's not there already).  Validate Joomla functions on-par when running PDO PostgreSQL as it relates to the native PostgreSQL driver.

### Documentation Changes Required

Update 4.0 compatibility references.
